lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -106,11 +106,11 @@ void GUIUtils::displayCenterBig(String msg) {
 #else
     if (dw > 64) {
         u8g2.setCursor(dw - 64, 6);
-        u8g2.setFont(u8g2_font_inb24_mn);
+        u8g2.setFont(u8g2_font_inb19_mn);
```

Comment:
I think that maybe is better check the characters size of `msg` and with that take the decision for change the font. What do you think?

Reply:
ok, done with deviceType